src/components/Navbar.module.css

```diff
@@ -1,32 +1,136 @@
 @import url("https://fonts.googleapis.com/css2?family=Inter:wght@300;600&family=Maven+Pro:wght@500&display=swap");
+.coName {
+  flex: 2;
+  font-size: 30px;
+}
 
-.logo {
-  margin-top: 2px;
-  margin-left: -20px;
+.logoContainer {
+  display: flex;
+  flex: 0.75;
+  justify-content: center;
+  align-items: center;
 }
 
 .logoAndName {
   display: flex;
-  align-items: center;
-  gap: 50px;
-  font-size: 35px;
+  flex: 3.2;
 }
 
 .navbar {
   display: flex;
   align-items: center;
-  justify-content: space-around;
-  gap: 100px;
   background-color: rgb(0, 62, 82);
-
   color: white;
-  height: 70px;
+  height: 3em;
   font: bolder 25px "Maven Pro", sans-serif;
 }
 
 .navLinks {
   display: flex;
-  gap: 50px;
+  flex: 7;
+  justify-content: space-evenly;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 10px;
+}
+
+@media screen and (max-width: 1008px) {
+  .navLinks {
+    display: none;
+  }
+
+  .hamburgerMenuContainer {
+    border: white 1px solid;
+    border-radius: 5px;
+    padding: 20px 8px;
+    cursor: pointer;
+  }
+
+  .hamburgerMenuIcon,
+  .hamburgerMenuIconOpen {
+    position: relative;
+    width: 35px;
+    height: 5px;
+    background: white;
+    border-radius: 5px;
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
+  }
+
+  .hamburgerMenuIcon::before,
+  .hamburgerMenuIcon::after,
+  .hamburgerMenuIconOpen::before,
+  .hamburgerMenuIconOpen::after {
+    content: "";
+    position: absolute;
+    width: 35px;
+    height: 5px;
+    background: white;
+    border-radius: 5px;
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
+    transition: all 0 ease-in-out;
+  }
+
+  .hamburgerMenuIcon::before,
+  .hamburgerMenuIconOpen::before {
+    transform: translateY(-12px);
+  }
+
+  .hamburgerMenuIcon::after,
+  .hamburgerMenuIconOpen::after {
+    transform: translateY(12px);
+  }
+
+  .hamburgerMenuContainer .hamburgerMenuIconOpen {
+    transform: translateX(-50px);
+    background: transparent;
+    box-shadow: none;
+  }
+
+  .hamburgerMenuIconOpen::before {
+    transform: rotate(45deg) translate(35px, -35px);
+  }
+
+  .hamburgerMenuIconOpen::after {
+    transform: rotate(-45deg) translate(35px, 35px);
+  }
+
+  .dropdown {
+    position: absolute;
+    background-color: white;
+    display: flex;
+    width: 100%;
+    justify-content: space-evenly;
+    align-items: center;
+    opacity: 0.8;
+    height: 3em;
+    background-color: rgb(0, 62, 82);
+    z-index: 1;
+    width: calc(100% - 40px);
+    font: bolder 20px "Maven Pro", sans-serif;
+  }
+
+  .hr {
+    margin: 0;
+  }
+
+  .dropdownLinks {
+    text-decoration: none;
+    color: white;
+  }
+}
+
+@media screen and (min-width: 1009px) {
+  .hamburgerMenuIcon {
+    display: none;
+  }
+
+  .hr {
+    display: none;
+  }
+
+  .dropdown {
+    display: none;
```

Comment:
this could be shortened to 

.hamburgerMenuIcon,
.hr,
.dropdown {
    display: none;
}